components/banner/browser-is-outdated.tsx

```diff
@@ -32,12 +32,29 @@ export const BrowserIsOutdatedBanner: React.FC<{}> = () => (
         </p>
       </div>
     </div>
+
+    <script
+      //   Script that shows warning for browser without ESModule
+      //  https://caniuse.com/?search=esmodule
+      //  https://stackoverflow.com/questions/44891421/detect-es6-import-compatibility
+      dangerouslySetInnerHTML={{
+        __html: `
+          const esModuleDisabled = typeof window !== 'undefined' 
+            && !('noModule' in HTMLScriptElement.prototype)
+            
+          if (esModuleDisabled) {
+            getElementById('browser-is-outdated').style.display = 'block'
+          }
```

Comment:
Not sure `const`, `&&` and `style.display` is backward compatible with very old browsers. At least we should encapsulate iin a try catch that would default on displaying the banner

Reply:
I think we're safe for && and style
- https://caniuse.com/mdn-html_elements_style
- https://caniuse.com/mdn-javascript_operators_logical_and

I can replace const for var to allow IE < 11